src/pkg/utils/wait.go

```diff
@@ -28,6 +36,49 @@ func isJSONPathWaitType(condition string) bool {
 	return true
 }
 
+func ExecuteWaitResource(ctx context.Context, gk schema.GroupKind, namespace, name string) error {
+	cfg, err := config.GetConfig()
+	if err != nil {
+		return err
+	}
+	mgr, err := ctrl.NewManager(cfg, ctrl.Options{})
+	if err != nil {
+		return err
+	}
+	identifiers := []object.ObjMetadata{
+		{
+			GroupKind: gk,
```

Comment:
Group has to be set for Kinds that have one. I nice thing to do would be to resolve the group for the user.

Reply:
Resolve this at a later time.